recipe/meta.yaml

```diff
@@ -25,31 +25,31 @@ source:
     - no-cocoa-notification.patch  # [osx]
 
 build:
-  number: 3
+  number: 4
   skip: true  # [win]
   detect_binary_files_with_prefix: true
 
 requirements:
   build:
     - python
-    - xz
```

Comment:
I had thought xz was needed here for something downstream, but looking at this again it seems like I am mistaken. https://developer.gnome.org/glib/stable/glib-building.html